diff.py

```diff
@@ -2129,8 +2133,12 @@ class Diff:
     max_score: int
 
 
-def is_nops(lines: List[Line]) -> bool:
-    return all(line.mnemonic == "nop" for line in lines)
+def get_trim_point(lines: List[Line], arch_settings: ArchSettings) -> bool:
```

Comment:
this won't type-check! (run `mypy`) But also, I think `lines = trim_nops(lines, arch)` would be a clearer API, "trim point" is a bit vague. arch_settings -> arch for consistency